### PR TITLE
Add Initial Flags blog post and flags to navigation of abseil.io

### PR DESCRIPTION
**Pull request by manshreck**

### CODE REVIEW

File: _posts/2019-05-09-flags-library.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+title: "Abseil Flags"
+layout: blog
+sidenav: side-nav-blog.html
+published: true
+permalink: blog/20190509-flags
+type: markdown
+category: blog
+excerpt_separator: <!--break-->
+---
+By [Gennadiy Rozenthal](mailto:rogeef@google.com), Abseil Engineer
```
Comment (derekmauro) [inline _posts/2019-05-09-flags-library.md]: rogeeff

File: _posts/2019-05-09-flags-library.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+* `int64_t`
+* `uint64_t`
+* `double`
+* `std::string`
+* `std::vector<std::string>`
+* `absl::Duration`
+* `absl::Time`
+
+For more information, consult the
+[Abseil Flags library documentation][abseil-flags].
```
Comment (derekmauro) [inline _posts/2019-05-09-flags-library.md]: Is this correct? This doesn't look like the right way to make a link in markdown.

File: _posts/2019-05-09-flags-library.md (hunk 1 above)
Comment (manshreck) [reply _posts/2019-05-09-flags-library.md]: Fixed

File: _posts/2019-05-09-flags-library.md (hunk 2 above)
Comment (manshreck) [reply _posts/2019-05-09-flags-library.md]: Yes, it's correct. It declared a shortcut (defined at the bottom)

**Pull request merged**